src/Aspire.Hosting/Orchestrator/ApplicationOrchestrator.cs

```diff
@@ -327,7 +327,7 @@ void InitializeConnectionString(ConnectionStringResource connectionStringResourc
                 await PublishConnectionStringAvailableEvent(connectionStringResource, cancellationToken).ConfigureAwait(false);
                 await _notificationService.PublishUpdateAsync(connectionStringResource, s => s with
                 {
-                    State = new(KnownResourceStates.Active, KnownResourceStateStyles.Info),
+                    State = new(KnownResourceStates.Active, KnownResourceStateStyles.Success),
```

Comment:
was this just a mistake previously?

Reply:
Yes, I changed the previous ones to Success in this PR https://github.com/dotnet/aspire/pull/10235 for parameters.